### ISSUE TITLE
Improve doc page readability

### ISSUE THREAD

**Issue by Evpok** [pytorch/pytorch#11968]
While the new documentation style looks sleeker, I find its readability awful

![image](https://user-images.githubusercontent.com/1656541/45916067-4c80bf80-be4f-11e8-9780-f8feaecafd66.png)

I can barely read the text in the grey-on-grey left navbar and the text of the main content is much too thin to be read comfortably.

cc @ezyang @gchanan @zou3519 @bdhirsh @jbschlosser @anjali411 @brianjo @mruberry

**Comment by m7thon** [comment]
As a workaround, cloning pytorch, and then doing:
```sh
cd docs
git checkout tags/v0.4.1 requirements.txt source/conf.py
pip install -r requirements.txt
make html
```
gives warnings, but does generate a readable version of the current docs with the old theme in  `docs/build/html`

**Comment by soumith** [comment]
this was fixed last week. Please let us know if the improvements look better.
- font was changed from a "thin" one to a more readable one
- vertical spacing was compressed in a few places
- general contrast was improved

**Comment by leomao** [comment]
I think the docs is more readable now.  But there are still some minor issues:
- some latex formulas are not rendered because of syntax errors. (there are some KaTeX errors reported in the browser console)
- inline code was segmented strangely. (ex: `torch.backends.cudnn.deterministic = True`)
![screenshot from 2018-10-07 10-52-41](https://user-images.githubusercontent.com/1811303/46577736-d324c780-ca1f-11e8-921d-0364ce82563f.png)
- It is not intuitive that the text `Search Docs` in the search docs box won't disappear when users focus on the box. Also, the search docs box doesn't look like an input field at the first glance.
- The orange dot at the left side of the current section is not align with the text properly.
![screenshot from 2018-10-07 11-06-08](https://user-images.githubusercontent.com/1811303/46577785-06b42180-ca21-11e8-9308-92d0d3381b8c.png)

**Comment by noeagles** [comment]
- the font is still not that readable as it used to be
- the gray background color of the left side of a page is still confusing; it should be black.

**Comment by m7thon** [comment]
Well, readability has been improved, yes, but it is still not as good as the old style. Here are some issues:
* The font for text in the new style is smaller than in the old style (and still thinner with less contrast), while the distance between lines is larger. Also, the grey boxes surrounding function names are now quite large, and vertical spacing in general is quite large. Furthermore, the menu bar is now shown on top at all times. The result is:
    - Harder to read text
    - Very spread out look
    - The old style fit about 50% more information on the screen at a time (less scrolling around)
* The menu items on the sidebar no longer expand (are not nested). This used to be extremely helpful to quickly browse what is available. I see that there is now a new sidebar on the right if you make the window wide enough, which shows fully expanded contents, but this is not very useful.
    - while you are at it, it would be nice to have the left side-bar *not* scroll with the main content

**Comment by meeotch** [comment]
+1 for expandable items on the left sidebar.

Also, I used to be able to read & navigate the docs (with sidebar) in half of my screen, with an open jupyter notebook in the other half.  Now, both sidebars disappear at half-screen window size, and I'm forced to ctrl-g Find my way through a giant page of everything jammed together.  (With attendant performance problems on a slow laptop, and the inability to search for things that may be at the end of what is a very long page load.)  My memory of the old docs is that google search would take you directly to the term you searched for, with no waiting, and you could easily look at related terms (say, loss functions), because the sidebar would be opened to that section.

**Comment by ezyang** [comment]
Many of the issues in this issue have still not been resolved. (I checked up on Search Docs problem and orange dot problem and they both reproed)

**Comment by brianjo** [comment]
@woo-kim Woo, I think we need to get Shift to take a look at these two issues. (Red dot isn't centered on line, and "Search Docs" text remaining in the search box with the cursor is in the box.) Thanks!

**Comment by rodrigoberriel** [comment]
As I mentioned in https://github.com/pytorch/pytorch_sphinx_theme/pull/147, it seems that the "red/oragen dot alignment" problem was fixed in https://github.com/pytorch/pytorch_sphinx_theme/commit/374627a5cac62ee9f9cb2762c63d1ecd37f154e7 (at least, it is aligned in my browser), so I submitted a PR there to fix the "search placeholder not hiding" problem. If it is merged, will there be anything left to do here @ezyang?

**Comment by ezyang** [comment]
@rodrigoberriel well, that would cover the ones I spot checked in March. I'd recommend doing a final walkthrough on the docs for all of the issues mentioned here and double checking before closing.

**Comment by svekars** [comment]
It looks like the problems described in this thread are resolved. Closing this - if there is more to do, feel free to reopen.
